Templace_I2C/Template_I2C_Basic.ino

```diff
@@ -0,0 +1,29 @@
+#include <Wire.h>
+
+int Address = 8;
+
+void setup() {
+  Wire.begin(Address);           // join i2c bus with address #8
+  Wire.onRequest(requestEvent); // register event
+}
+
+void loop() {
+  delay(100);
+}
+
+// function that executes whenever data is requested by master
+// this function is registered as an event, see setup()
+void requestEvent() {
+  union floatToBytes {
+    char buffer[4];
+    float voltageReading;
+  } converter;
+  
+  converter.voltageReading = getReading0(); //sensor reading here
```

Comment:
The client should be reading a byte from the brain here. That will tell it which sensor reading to call, 
Example,
```
int sensorNum = Wire.read();
if(sensorNum == 0){
   converter.sensorReading = getReading0();
} else if (sensorNum == 1){
    converter.sensorReading = getReading1();
}
```

Reply:
Got it will address this!